src/plugin.json

```diff
@@ -1,17 +1,24 @@
 {
+  "$schema": "https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json",
   "type": "app",
   "name": "Simple App",
-  "id": "yourorg-simple-app",
+  "id": "myorgid-simple-app",
 
   "info": {
+    "description": "Grafana Panel Plugin Template",
     "author": {
-      "name": "Grafana Project",
-      "url": "https://grafana.com"
+      "name": "Your Name"
     },
+    "keywords": ["panel", "template"],
     "logos": {
-      "small": "img/logo.png",
-      "large": "img/logo.png"
+      "small": "img/logo.svg",
+      "large": "img/logo.svg"
     },
+    "links": [
+      { "name": "Website", "url": "https://github.com/grafana/grafana-starter-panel" },
```

Comment:
```suggestion
      { "name": "Website", "url": "https://github.com/grafana/simple-app-plugin" },
```

Reply:
Probably better to rename this to `grafana-starter-app` right away, even if we haven't renamed the repo yet.

Reply:
True, didn't know about your plans to rename the repo but makes sense. :) :+1:

Reply:
Gotcha. Yeah, we decided to adopt a naming convention for all starters, so it would be great if this repository could follow that as well.

Reply:
I've updated my suggestions.